scipy/signal/spectral.py

```diff
@@ -1787,3 +1801,27 @@ def _triage_segments(window, nperseg,input_length):
                                  " length of window")
     return win, nperseg
 
+
+def _median_bias(n):
+    """
+    Returns the bias of the median of a set of periodograms relative to
+    the mean.
+
+    See arXiv:gr-qc/0509116 Appendix B for details.
+
+    Parameters
+    ----------
+    n : int
+        Numbers of periodograms being averaged.
+
+    Returns
+    -------
+    bias : float
+        Calculated bias.
+    """
+    if n >= 1000:  # large n limit
+        return log(2)
+    bias = 1
+    for i in xrange(1, int((n-1)/2) + 1):
```

Comment:
better to do `(n-1) // 2 + 1` (all int arithmetic)

Also it's probably cleaner to vectorize (allocs here will be a tiny cost) rather than `xrange`:
```
ii_2 = 2 * np.arange(1, (n-1) // 2 + 1)
bias = np.sum(1. / (ii_2 + 1) - 1. / ii_2)
```